src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java

```diff
@@ -982,6 +991,22 @@ public synchronized void setApprovedScriptHashes(String[] scriptHashes) throws I
         reconfigure();
     }
 
+    @DataBoundSetter
+    public synchronized void setForceSandbox(boolean forceSandbox) {
```

Comment:
Why is this `synchronized`? I think `save`  is already taking care of file system write synchronization, isn't it?

Reply:
I use it as it is already used in all the other @DataBoundSetter in this class,

```
public synchronized void setApprovedScriptHashes
public synchronized void setApprovedSignatures
```

 in case this is not required just let me know and I can remove

Reply:
These are writing to more complex fields that are read from other places, so write operations need to block other threads. I don't think setting a boolean which only written from a form save in the UI requires to be synchronized.

Reply:
Perfect, changed then